todo.md

```
@@ -53,14 +61,14 @@ This file tracks the concrete tasks required to execute the project roadmap.
     -   [ ] Integrate more deeply with libraries like `NDlib` (Network Diffusion Library).
     -   [ ] Implement spatial diffusion models that account for geographic distance (gravity models).
 
-## Phase 6: Heterogeneity & Segmentation (Future)
+## Phase 7: Heterogeneity & Segmentation (Future)
 -   [ ] **Latent-Class & Hierarchical Models**
-    -   [ ] Implement finite-mixture models (e.g., `MixtureBassModel`) to automatically infer adopter segments.
+    -   [ ] Implement finite-mixture models to automatically infer adopter segments.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)